src/views/ModelPlan/ReadOnly/GeneralCharacteristics/index.tsx

```diff
@@ -172,111 +213,151 @@ const ReadOnlyGeneralCharacteristics = ({
       </div>
 
-      <div className="margin-bottom-4 padding-bottom-2 border-bottom-1px border-base-light">
-        <div className="desktop:display-flex flex-justify">
-          <div className="desktop:width-card-lg">
-            <ReadOnlySection
-              heading={t('careCoordination')}
-              copy={translateBooleanOrNull(careCoordinationInvolved)}
-            />
-          </div>
-          {careCoordinationInvolved && (
-            <div className="desktop:width-card-lg">
-              <ReadOnlySection
-                heading={h('howSo')}
-                copy={careCoordinationInvolvedDescription}
-              />
-            </div>
-          )}
-        </div>
+      <div
+        className={`${
+          isViewingFilteredView
+            ? ''
+            : 'margin-bottom-4 border-bottom-1px border-base-light padding-bottom-2'
+        }`}
+      >
+        {checkGroupMap(
+          isViewingFilteredView,
+          filteredQuestions,
+          'careCoordination',
+          <SideBySideReadOnlySection
+            firstSection={{
+              heading: t('careCoordination'),
+              copy: translateBooleanOrNull(careCoordinationInvolved)
+            }}
+            secondSection={
+              careCoordinationInvolved === true && {
+                heading: h('howSo'),
+                copy: careCoordinationInvolvedDescription
+              }
+            }
+          />
+        )}
         {careCoordinationInvolvedNote && (
           <ReadOnlySection
             heading={t('basics:notes')}
             copy={careCoordinationInvolvedNote}
           />
         )}
 
-        <div className="desktop:display-flex flex-justify">
-          <div className="desktop:width-card-lg">
-            <ReadOnlySection
-              heading={t('additionalServices')}
-              copy={translateBooleanOrNull(additionalServicesInvolved)}
-            />
-          </div>
-          {additionalServicesInvolved && (
-            <div className="desktop:width-card-lg">
-              <ReadOnlySection
-                heading={h('howSo')}
-                copy={additionalServicesInvolvedDescription}
-              />
-            </div>
-          )}
-        </div>
+        {checkGroupMap(
+          isViewingFilteredView,
+          filteredQuestions,
+          'additionalServices',
+          <SideBySideReadOnlySection
+            firstSection={{
+              heading: t('additionalServices'),
+              copy: translateBooleanOrNull(additionalServicesInvolved)
+            }}
+            secondSection={
+              additionalServicesInvolved === true && {
+                heading: h('howSo'),
+                copy: additionalServicesInvolvedDescription
+              }
+            }
+          />
+        )}
         {additionalServicesInvolvedNote && (
           <ReadOnlySection
             heading={t('basics:notes')}
             copy={additionalServicesInvolvedNote}
           />
         )}
```

Comment:
Another instance where this component will always render in every view if a note exists